### DISCUSSION TITLE
Duplicate Fetching Changelog Messages

### DISCUSSION THREAD

**Discussion by jjathman** [renovatebot/renovate#8760] [Request Help] [answered]
I'm looking in to how to speed up the processing of some of our repositories with Renovate (local installation). One thing that appears to take a great deal of time is fetching change logs. With debug logging enabled, I see what appears to be the exact same message many times for every dependency. Do I have something configured incorrectly? Is it really not doing the exact same thing each time? Here's an example:

```
DEBUG: Fetching changelog: https://github.com/spring-projects/spring-framework (5.2.8.RELEASE -> 5.3.4) (repository=ma/xxx)
DEBUG: Fetching changelog: https://github.com/spring-projects/spring-framework (5.2.8.RELEASE -> 5.3.4) (repository=ma/xxx)
DEBUG: Fetching changelog: https://github.com/spring-projects/spring-framework (5.2.8.RELEASE -> 5.3.4) (repository=ma/xxx)
```

this might appear 50 times or more in the log. Just wondering if there's a way I can speed this up somehow?

**Comment by viceice** [comment, marked as answer]
The the lower level http requests are cached, so this shouldn't really matter

**Reply by jjathman** [reply]
Thanks, you are correct, it looks like all of those messages happen almost instantaneously with each other. Long story short I have a Java with Gradle repository that is taking more than 13 minutes to process completely. Just trying to see if I'm missing any sort of low hanging fruit for improving performance.

**Reply by rarkins** [reply]
Try disabling the gradle manager and enabling gradle-lite

**Reply by jjathman** [reply]
Sorry I'm not finding the documentation about that option. Can you point me to it? 

From what I can tell it isn't the Gradle part itself that is taking a long time, it is more the shear number of dependencies and how out of date those dependencies are I believe.

**Reply by rarkins** [reply]
What's the timing splits printed in the logs at the end of the run?

**Reply by jjathman** [reply]
> Try disabling the gradle manager and enabling gradle-lite

I ended up going with this and it worked great, thanks for the suggestion. Sorry it took me so long to respond.
